src/System.Management.Automation/engine/hostifaces/pipelinebase.cs

```diff
@@ -413,7 +413,7 @@ public override Collection<PSObject> Invoke(IEnumerable input)
 
             // Execution completed successfully
             // 2004/06/30-JonN was ReadAll() which was non-blocking
-            return Output.NonBlockingRead(Int32.MaxValue);
+            return Output.Read(Int32.MaxValue);
```

Comment:
Why do we change this?

Reply:
Sorry, this was a local change that I accidentally committed.